### DISCUSSION TITLE
Error with Puro Responsive sizer package

### DISCUSSION THREAD

**Discussion by alphacruze** [pingbird/puro#55] [Q&A]
I've been using puro recently and on a project where I did not have it active, somehow VS started referring to Puro caching

<img width="776" alt="image" src="https://github.com/pingbird/puro/assets/103413789/7bea27ec-1f9d-4156-a31d-53efffae6673">

Due to that I started experiencing the above issue where Responsive Sizer is no longer allowing me to run my app (through chrome at least) 

When I go to the directory and line specified I come across this, the "View" which is active in my other projects works perfectly, but just not in this repo
VS is also reading that I have over 10,000 files changed...

<img width="505" alt="image" src="https://github.com/pingbird/puro/assets/103413789/392d1222-a1fb-4ad6-a050-7808a68e4e87">

<img width="127" alt="image" src="https://github.com/pingbird/puro/assets/103413789/e755e1b9-ec93-4ad6-98ea-7f6bdf68e172">
The 10k changes git detecting

**Comment by pingbird** [comment]
It sounds like the version of flutter that you're building the project with is too old for the version of sizer that it depends on, you can use `puro use -g <name>` to switch the global default flutter version (VSCode should also pick up on it). Also be sure to run `flutter pub get` after changing the flutter version or any dependencies.

As for the git changes, did you accidentally create a git repository in your home folder? Not something Puro could do.

**Reply by alphacruze** [reply]
I have not made a new git repo, it is incredibly weird, I'll have to go look for info on that

Can't be that the package isn't compatible, I have been using the same version of the package for months with no problem, and even the latest version of it can work on dark sdk 2.12

i also have another repo with the same settings and it doesn't have that issue
